toyota_nodsu_pt_generated.dbc

```diff
@@ -403,14 +389,53 @@ BO_ 1592 DOOR_LOCKS: 8 XXX
  SG_ LOCK_STATUS : 20|1@0+ (1,0) [0|1] "" XXX
  SG_ LOCKED_VIA_KEYFOB : 23|1@0+ (1,0) [0|1] "" XXX
 
+BO_ 865 CLUTCH: 8 XXX
+ SG_ CLUTCH_PRESSED : 38|1@0+ (1,0) [0|1] "" XXX
```

Comment:
add this to `opendbc\generator\toyota\_toyota_2017.dbc`, don't change the rest of the file, and run `generator.py` with python